sdcm/cluster.py

```diff
@@ -302,41 +301,57 @@ def init(self) -> None:
                              exc_info=True)
         self._init_argus_resource()
 
+    @cached_property
+    def argus_client(self) -> ArgusSCTClient | None:
+        try:
+            run_id = self.test_config.test_id()
+            creds = KeyStore().get_argus_rest_credentials()
+            client = ArgusSCTClient(run_id=run_id, auth_token=creds["token"], base_url=creds["baseUrl"])
+            return client
+        except Exception:  # pylint: disable=broad-except
+            LOGGER.error("Error creating argus client", exc_info=True)
+        return None
+
     def _init_argus_resource(self):
```

Comment:
might consider rename this one as well, this doesn't init the resource
it's indicating start of a test

Reply:
Done